apps/condo/domains/billing/access/SearchBillingReceiptsWithoutConsumerService.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Generated by `createservice billing.SearchBillingReceiptsWithoutConsumerService --type queries`
+ */
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+
+async function canSearchBillingReceiptsWithoutConsumer ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    return true
```

Comment:
O-o!

Reply:
any non-deleted user will be able to read these... are we sure about that ?

Reply:
As discussed with @pahaz we are going to responds with account data only